sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -1690,11 +1743,14 @@ public new DataLakeFileClient WithCustomerProvidedKey(Models.DataLakeCustomerPro
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable AZC0002
```

Comment:
I think the disable and restore AZC0002 can be removed.

Reply:
it complains unfortunately, will keep it